auth/config.go

```diff
@@ -0,0 +1,172 @@
+package auth
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+const (
+	constMfa                      = "mfa"
+	constMultiFactorConfig        = "multiFactorConfig"
+	constProviderConfigs          = "providerConfigs"
+	constTotpProviderConfig       = "totpProviderConfig"
+	constAdjacentIntervals        = "adjacentIntervals"
+	constMfaObject                = "MultiFactorConfig"
+	constProviderConfigObject     = "ProviderConfig"
+	constTotpProviderConfigObject = "TotpProviderConfig"
+	constState                    = "state"
+	constEnabledProviders         = "enabledProviders"
+	constFactorIds                = "factorIds"
+)
+
+var validAuthFactors = map[string]bool{
+	"PHONE_SMS": true,
```

Comment:
Should this include SMS or TOTP?

Reply:
SMS only, we've switch to `providerConfigs` for factors other than SMS.